plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/shipping-rates-control-package/index.tsx

```diff
@@ -137,22 +120,7 @@ export const ShippingRatesControlPackage = ( {
 	}
 
 	return (
-		<div
-			className={ clsx(
-				'wc-block-components-shipping-rates-control__package',
-				className,
-				{
-					'wc-block-components-shipping-rates-control__package--disabled':
-						isSelectingRate,
-					'wc-block-components-shipping-rates-control__package--first-selected':
-						! isSelectingRate && selectedOptionNumber === 0,
-					'wc-block-components-shipping-rates-control__package--last-selected':
-						! isSelectingRate &&
-						selectedOptionNumber ===
-							packageData?.shipping_rates?.length - 1,
```

Comment:
I am not sure whether we should be worried about it being a potentially breaking change. If so, we can steer folks to use classes in the underlying `RadioControl` component that serve a similar purpose https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/client/blocks/packages/components/radio-control/index.tsx#L41-L45